Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -102,44 +105,38 @@ class BTAnalyticsService: Equatable {
         )
 
         await BTAnalyticsService.events.append(event)
-        
-        do {
-            let configuration = try await apiClient.fetchConfiguration()
```

Comment:
This `apiClient.fetchConfiguration()` isn't used anymore, if we want to delete it entirely